docs/pages/workflow/upgrading-expo-sdk-walkthrough.mdx

```diff
@@ -3,96 +3,170 @@ title: Upgrade Expo SDK
 description: Learn how to incrementally upgrade the Expo SDK version in your project.
 ---
 
+import { Terminal } from '~/ui/components/Snippet';
+import { Step } from '~/ui/components/Step';
+import { Tabs, Tab } from '~/ui/components/Tabs';
+
 > **info** We recommend upgrading SDK versions incrementally, one at a time. Doing so will help you pinpoint breakages and issues that arise during the upgrade process.
 
-Expo maintains ~6 months of backward compatibility. Once an SDK version has been deprecated, you will no longer be able to use the Expo Go app. However, you will still be able to publish updates via `eas update`, and build using `eas build`. Deprecations **will not** affect the standalone apps you have in production.
+Expo maintains ~6 months of backward compatibility. Once an SDK version has been deprecated, you will no longer be able to use it in the Expo Go app. SDK deprecations **will not** affect the standalone apps you have in production or your [development builds](/develop/development-builds/introduction/), and backwards compatibility for older SDK versions on EAS services tends to be much longer, but not forever.
```

Comment:
```suggestion
Expo maintains approximately 6 months of backward compatibility. Once an SDK version has been deprecated, you can no longer use it in the Expo Go app. SDK deprecations **will not** affect the standalone apps you have in production or your [development builds](/develop/development-builds/introduction/), and backwards compatibility for older SDK versions on EAS services tends to be much longer, but not forever.
```

- First change is unrelated to this PR but we should word out instead of using a tilde. Not good for screen readers, etc.